packages/loot-core/migrations/1688841238000_add_account_type.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE accounts ADD COLUMN type INTEGER;
```

Comment:
Wasn’t it a string before? This might still break sync since the message includes the account type as part of its value.